TERMSOFSERVICE.md

```diff
@@ -420,64 +457,68 @@ reasonable use, we have the right to block access to Tideways until such
 default has been rectified. In such cases we will normally notify you of such
 blocking. Our other legal and contractual rights shall remain unaffected.
 
-# 17. Change in ToS
+# 17. Changes to ToS
 
-17.1. We reserve the right to change the ToS at any time with effect from a
-future date. A change in essential provisions which influence your contractual
-claims shall be made only for valid and objective reasons, especially of a
-legal, technical and economic nature.
+17.1. Tideways reserves the right to change the ToS with an effective date in
+the future. A change in essential paragraphs that effect your contractual terms
+shall only be made because of valid and objective concerns, especially
+technical, legal and business reasons.
 
-17.2. We shall advise you of the amended ToS, at least in text form, so that
-you will have at least two weeks time to object to the change. In case of an
-objection, we can determine whether the previous ToS in our contractual
-relationship with you shall remain valid or whether you shall receive the right
-to termination. Should you not object to the amended ToS within the objection
-period, they shall be considered to have been accepted.
+17.2. Tideways will inform you about changes in the ToS in textual form, so
+that you have at least two (2) weeks time to object to the changes. In case of
+an objection Tideways will determine if the previous ToS will stay valid in the
+contractual agreement between both parties or if you are granted a right to
+termination of the contract. In case you don’t object to the changed ToS within
+the period for objection they count as accepted.
 
 # 18. Confidentiality
 
-18.1. The contracting parties undertake to treat all information they gain
-knowledge of during the fulfillment of this contract as confidential and to use
-it only for contractually agreed purposes.
-
-18.2. The rights and obligations in accordance with this section concerning
+18.1. Protected or confidential information of the contracting parties for the
```

Comment:
Modified Confidentiality clauses to more robust wording that protects both parties (you and us) from breaches in confidential information. Added exception clauses in § 18 (2) which have been missing before but are standard procedure in most ToS.